src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-sun calendar--month-length-31">
```

Comment:
Consider generating the days of the calendar dynamically using a template engine or JavaScript, as manually adding each day can be error-prone and not scalable.